package.json

```diff
@@ -596,7 +597,9 @@
         "download-edge": "node scripts/downloadAndExtractEdge.js"
     },
     "dependencies": {
+        "bufferutil": "4.0.1",
         "puppeteer-core": "9.1.1",
+        "utf-8-validate": "5.0.2",

```

Comment:
These were needed for me to be able to create a VSIX. Otherwise it failed in an npm step

Reply:
seems to be associated with the websocket so importing them should be fine, although I'm not sure why they were not required before as I don't see any new code bringing them in 😕. Could it be node version?

Reply:
Yeah, The error message I was seeing was these were marked as 'peer' dependencies of ws, which means I needed to manually install. LMK if I should remove from this PR, happy to do that

Reply:
Going to land as is since these are required for running `npm run package` for me. We can follow up if we decide to remove later